source/Xunit.Gherkin.Quick.ProjectConsumer/Placeholders/Placeholders.cs

```diff
@@ -44,10 +44,8 @@ public void Given_I_Have_Supplied_A_DocString_With(DocString data)
             var dataContent = _stringData.Content;
 
             Assert.NotNull(dataContent);
-            Assert.DoesNotContain('<', dataContent);
-            Assert.DoesNotContain('>', dataContent);
 
-            var expectedContent = $@"{date} Delicious {meal} plan:{Environment.NewLine}The main dish for {date} is: {mainDish}!{Environment.NewLine}Additionally prepare a {fruit}-based smoothie with some {addition}.{Environment.NewLine}Optionally add a cup of hot {hotDrink}? {needHotDrink}.{Environment.NewLine}Total portions needed: {myPortions}+{otherPortions}+{provision}!";
+            var expectedContent = $@"{date} Delicious {meal} plan:{Environment.NewLine}The main dish for {date} is: {mainDish}!{Environment.NewLine}Additionally prepare a {fruit}-based smoothie with some {addition}.{Environment.NewLine}Optionally add a cup of hot {hotDrink}? {needHotDrink}.{Environment.NewLine}Total portions needed: {myPortions}+{otherPortions}+{provision}!{Environment.NewLine}<--- Hooray! {meal} is done! --->";
```

Comment:
Added a simple case for DocString's placeholder to highlight this behavior in these tests as well. More thorough tests are in GherkinScenarioOutlineTests file.